src/django/api/models/utility.py

```diff
@@ -1,16 +1,25 @@
 from django.db import models
 from django.contrib.gis.db import models as gis_models
 from django.contrib.gis.geos import Point
+from .state import State
 
 RALEIGH = Point(-78.6382, 35.7796)
 
+
+def get_default_state():
+    return State.objects.get(id="NC").id
```

Comment:
This is tautological, since we're asking for the `id` of the row with `id="NC"`. Let's just do:

```python
    state = models.ForeignKey(
        State, on_delete=models.PROTECT, default="NC"
    )
```